app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -423,7 +424,7 @@ public function buildOnCheckCallback($checkoutType, $quote)
                         } "). "
         
                         bolt_hidden.classList.add('required-entry');
-                        return is_valid;
+                        return (isAdminFormValid = is_valid);
```

Comment:
What is this assignment used for? Do you mean to do the assignment here? It works fine since it's returning `is_valid`, but this is a little misleading and hard to read.